test/kinematic/tangential_acceleration_of_rotating_body_test.py

```diff
@@ -0,0 +1,45 @@
+from collections import namedtuple
+from pytest import approx, fixture, raises
+from symplyphysics import (
+    errors,
+    units,
+    convert_to,
+    Quantity,
+    SI,
+)
+from symplyphysics.laws.kinematic import tangential_acceleration_of_rotating_body as tangential_acceleration_law
+
+# Description
+## A body is moving along a curve. Its angular acceleration is 3.5 rad/s^2 and its rotation radius
+## about the momentary rotation axis is 0.1 m. The magnitude of its tangential acceleration should
```

Comment:
```suggestion
## around the momentary rotation axis is 0.1 m. The magnitude of its tangential acceleration should
```

Reply:
![image](https://github.com/blackyblack/symplyphysics/assets/115647936/8bc9289b-cdcc-4c9c-b1e2-e8233ddb32db)
_about_ is also possible in this context

Reply:
Thanks. You could leave it this way as you've proven your point. No need to blindly follow my review :)

Reply:
Oh don't worry, I actually figured the other option was better!